content/articles/heroku-error-differentapp.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: Troubleshooting Heroku shows different app error
+excerpt: You pointed your domain to Heroku, but when you access your domain you see a different application than the one you expected.
+categories:
+- Heroku and DNSimple
+---
+
+# Troubleshooting Heroku shows different app error
+
+You pointed your domain to Heroku, but when you access your domain from the browser you see a different application than the one you expected.
+
+This error occurs when you point a custom domain to Heroku, but there is another app at Heroku with that domain name set up as a custom domain.

```

Comment:
This sentence is a little bit unclear. I'd rephrase it to

> This error occurs when in DNSimple you point the DNS for a domain to Heroku, but in Heroku you associated the custom domains to a different application.

I'd also add an example

> For example, you have two Heroku applications: `green-turtle` and `red-turtle` and the corresponding Heroku hostnames `green-turtle.herokuapp.com` and `red-turtle.herokuapp.com`. You add the domain `awesome-turtles.com` in DNSimple and you point the CNAME `www.awesome-turtles.com` to `green-turtle.herokuapp.com`. However, in Heroku you associate the `www.awesome-turtles.com` custom domain to the application `red-turtle.herokuapp.com`.
> What will happen is that when someone will go to `www.awesome-turtles.com`, Heroku will route the request to the `red-turtle` app, and not `green-turtle` as you would expect.